config/jobs/kubernetes-sigs/testing_frameworks/testing-frameworks-config.yaml

```diff
@@ -7,13 +7,35 @@ presubmits:
     skip_report: false
     labels:
       preset-service-account: "true"
+      preset-dind-enabled: "true"
```

Comment:
Why do you need to use docker-in-docker?

Reply:
Essentially our tests want a local docker engine to talk to.